packages/flutter_tools/lib/src/build_runner/devfs_web.dart

```diff
@@ -53,8 +53,9 @@ class WebAssetServer implements AssetReader {
     UrlTunneller urlTunneller,
     BuildMode buildMode,
     bool enableDwds,
-    Uri entrypoint,
-  ) async {
+    Uri entrypoint, {
+    bool testMode = false,
```

Comment:
Could you explain what `testMode` means?

Reply:
Added doc comments